samcli/lib/list/mapper.py

```diff
@@ -0,0 +1,13 @@
+"""
+Interface for Mappers
+"""
+import abc
+from typing import Generic, TypeVar
+
+T = TypeVar("T")
+
+
+class Mapper(Generic[T]):
+    @abc.abstractmethod
+    def map(self, data: T):
+        pass
```

Comment:
Instead of `pass` for these abstract methods, let's add docstrings.

Reply:
Added the docstring